api/src/main/java/uk/gov/dhsc/htbhf/eligibility/service/EligibilityService.java

```diff
@@ -79,10 +83,19 @@ private HMRCEligibilityRequest createHMRCRequest(PersonDTO person, LocalDate cur
 
     private EligibilityResponse buildEligibilityResponse(DWPEligibilityResponse dwpEligibilityResponse,
                                                          HMRCEligibilityResponse hmrcEligibilityResponse) {
-        return EligibilityResponse.builder()
-                .eligibilityStatus(statusCalculator.determineStatus(dwpEligibilityResponse, hmrcEligibilityResponse))
-                .dwpHouseholdIdentifier(dwpEligibilityResponse.getHouseholdIdentifier())
-                .hmrcHouseholdIdentifier(hmrcEligibilityResponse.getHouseholdIdentifier())
-                .build();
+
+        EligibilityResponse.EligibilityResponseBuilder builder = EligibilityResponse.builder()
+                .eligibilityStatus(statusCalculator.determineStatus(dwpEligibilityResponse, hmrcEligibilityResponse));
+
+        if (EligibilityStatus.ELIGIBLE == dwpEligibilityResponse.getEligibilityStatus()) {
```

Comment:
minor comment, but this might be more readable as `dwpEligibilityResponse.getEligibilityStatus() == ELIGIBLE`

Reply:
Done.